### ISSUE TITLE
GCWeb: Splash page - Canada.ca assets down

### ISSUE THREAD

**Issue by DamonU2** [wet-boew/GCWeb#2033]
The working example and splash page example links both return 404, as well as the assets (at least https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/wmms-spl.svg and https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/sig-spl.svg)

**Comment by anthonyfok** [comment]
@DamonU2 Thank you for reporting this issue!  Like you, I wish GCWeb has an easier way of maintaining backward compatibility and all, but anyhow...

I took a look at recent releases, and I think I found the "culprit" in Release v11.5.0 (<https://github.com/wet-boew/GCWeb/releases/tag/v11.5.0>) from 29 Sept 2022:

> ### Templates
> 
> * Minor - Splash page - Create French version + assets cleanup
>    Note: A communication about the changes is going to be sent to the departments
>
> ...
>
> ### Modified files for implementation
>
> * ...
> * (removed) GCWeb/assets/sig-spl.png
> * (removed) GCWeb/assets/sig-spl.svg
> * (removed) GCWeb/assets/wmms-spl.png
> * (removed) GCWeb/assets/wmms-spl.svg

(I wonder if we are in one of the "departments" where a communication about the changes was sent to.  :wink:)

The above changes can be found in commit 63c3117855fbfb48ce007dcd7c21ab180ad2ced6 which merged from Pull Request #2024.  The diff for templates/splash/layouts/splashpage-en.html and templates/splash/layouts/splashpage-fr.html does show something useful though:

```diff
<img src="{{ setting-resourcesBasePathTheme }}/assets/sig-spl.svg" width="283" alt="Government of Canada" property="logo" /><span class="wb-inv"> / <span lang="fr">Gouvernement du Canada</span></span>
<img src="{{ setting-resourcesBasePathTheme }}/assets/sig-blk-en.svg" width="283" alt="Government of Canada" property="logo" /><span class="wb-inv"> / <span lang="fr">Gouvernement du Canada</span></span>
```

```diff
<img src="{{ setting-resourcesBasePathTheme }}/assets/wmms-spl.svg" width="127" alt="Symbol of the Government of Canada" /><span class="wb-inv"> / <span lang="fr">Symbole du gouvernement du Canada</span></span>
<img src="{{ setting-resourcesBasePathTheme }}/assets/wmms-blk.svg" width="127" alt="Symbol of the Government of Canada" /><span class="wb-inv"> / <span lang="fr">Symbole du gouvernement du Canada</span></span>
```

In summary:

1. `https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/sig-spl.svg` becomes

    * https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/sig-blk-en.svg
        
        ![sig-blk-en.svg](https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/sig-blk-en.svg)

    * https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/sig-blk-fr.svg
        
      ![sig-blk-fr.svg](https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/sig-blk-fr.svg)

2. `https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/wmms-spl.svg` becomes

    * https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/wmms-blk.svg
    
        ![wmms-blk.svg](https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/wmms-blk.svg)

**Comment by duboisp** [comment]
@DamonU2 and @anthonyfok can you send us an email to ESDC.SD.DEV-DEV.DS.EDSC@servicecanada.gc.ca so I will be able to add your name into the message we are trying to send with more details about that breaking change and the we are on our way to prepare another one with the changes to make into the site footer. 

Side note: You can see more details in our recent draft to update for our [public versioning API (DD-14)](https://wet-boew.github.io/wet-boew-documentation/decision/14.html)

**Comment by duboisp** [comment]
@DamonU2 we re-added the splash page images into our distribution zip file. The previous releases have back that assets included.

We are now going to include in our versioning API files assets required after the implementation of GCWeb (sites, templates, components), like the splash page images. 

We are going to mark those assets as deprecated and it will remains in our distribution for at least one year to allow department to migrate.
